### DISCUSSION TITLE
Program or select a controller's LED number or color through web interface

### DISCUSSION THREAD

**Discussion by VR-Nima** [darthcloud/BlueRetro#467] [Ideas]
I purchased 4x BlueRetro adapters for N64 primarily to use with the Nintendo Switch Online Nintendo 64 controllers. Everything works great!

Except that every controller's LED shows as Player 1. Obviously, there's no way for the adapter to automatically identify which port it's in, but it would be really cool if you could manually select the player number through the web interface. This way, I'd be able to easily identify which controller is which (since the only color available is gray, there's not really a better solution except marking up the controller or adding a sticker).

This would be useful for lots of kinds of controllers, like DualShock 3 and Wii U Pro controllers.

For controllers like DualShock 4 and DualSense, allowing the user to select an LED color instead could be very useful.

**Comment by raed143** [comment]
It does work like this. If you have one board comnected to all ports. Try to get an adapter for 4 controller support or do one urself it's easy.

**Reply by VR-Nima** [reply]
That doesn't actually solve the problem. I want to have a toggle in the web tool that lets me program the controller's LED.

**Comment by darthcloud** [comment]
Someone could contribute this but it's not something I will personally work on.
